test/e2e_node/inode_eviction_test.go

```diff
@@ -141,165 +141,168 @@ func runEvictionTest(f *framework.Framework, testCondition string, podTestSpecs
 	Context(fmt.Sprintf("when we run containers that should cause %s", testCondition), func() {
 
 		tempSetCurrentKubeletConfig(f, updateFunction)
```

Comment:
Why not just move the function outside of current context? Less indent is preferred.